content/news/playtest-20160403.md

```diff
@@ -0,0 +1,118 @@
+We are pleased to announce the first playtest leading up to our next stable release.
+
+Our development branch currently contains more than five months worth of changes, many of them very fundamental. As such, we have finally entered the feature freeze phase of the development cycle, where we stop adding features and focus on fixing bugs found during playtesting. This is where *you* come into play! We need *your* help to give this playtest as wide a test coverage as possible to ensure another successful stable release of OpenRA! We therefore ask you to please report any and all bugs and problems you encounter on [our bug tracker](http://bugs.openra.net), in the OpenRA [IRC channel](/community/), or here in the comments. We will strive to fix as many bugs as possible before a final release. Thank you for helping to improve OpenRA!
+
+While there are the usual bug fixes and improvements, the biggest changes this time around are under the hood and will be felt foremost by server operators, modders and mappers. In fact, some of these changes are so fundamental that we will address them in-depth in the articles covering future playtests. To ensure that there will be enough dedicated servers available for playtesting, we will cover that topic below.
+
+Just to whet your appetite, dedicated servers no longer need to sync maps with the resource site, and it is now possible to add third-party mods as a single package (".oramod") to your installation.
+
+But one thing after the other...
+
+<hr />
+## Ingame changes
+<hr />
+
+Quite a few long-standing annoyances have been addressed during the last cycle. Just to name a few, players will no longer be kicked from the server after installing a map, walls now stop bullets reliably, and husks can be targeted on refineries.
+
+Other changes and new features include:
+
+* A couple of issues related to left-click controls have been solved.
```

Comment:
My experience is that people stop paying attention half way down the list, so can we put the most exciting things first?
IMO the order should be:
```
* By popular demand, a tech level without super-powers has been added to the Red Alert mod.
* Zooming is now possible by holding CTRL while scrolling your mouse wheel. For spectators and in the map editor, two additional zoom steps are available.
* Canceling an entire build queue at once is now possible by using CTRL + right click.
* Static defenses will now stop tracking their targets once it is out of range. This also sometimes prevented camo pillboxes from cloaking.
* Line-building of walls can now be disabled by holding the Shift key while placing the wall element.
* A couple of issues related to left-click controls have been solved.
* Engineers now have a lower selection priority so that they will not be selected along with your regular combat units.
* Fragile diplomacy has been removed.
* Lots of minor fixes, UI polish and performance improvements.
```